_pytest/main.py

```diff
@@ -763,7 +763,6 @@ def _matchnodes(self, matching, names):
                 if not has_matched and len(rep.result) == 1 and x.name == "()":
                     nextnames.insert(0, name)
                     resultnodes.extend(self.matchnodes([x], nextnames))
-            node.ihook.pytest_collectreport(report=rep)
```

Comment:
I'm wondering if this could break some expectations plugins had in some way? If so, maybe it should go to `features`?